### PR TITLE
completions: make `uuid -n` expect an argument

### LINKED ISSUE #636: completions: make `uuid -n` expect an argument

### PR DESCRIPTION
**Pull request by ee7**
Follow-up PR from #629.

@glennj does this look OK to you?

---

Before this commit, tab completion after `-n` would suggest global
options, even though `-n` without an argument is an error. For the fish
completions:

```console
$ configlet uuid -n[Tab]
-nh  (Show help)  -nt  (Select a track directory)  -nv  (Verbose level)
```

This commit removes that completion for both the bash and fish
completion scripts.

The fish [docs for `complete`](https://fishshell.com/docs/current/cmds/complete.html) describe the relevant options as:

> `-f` or `--no-files`
> This completion may not be followed by a filename.
> 
> `-r` or `--require-parameter`
> This completion must have an option argument, i.e. may not be followed by another option.
> 
> `-x` or `--exclusive`
> Short for `-r` and `-f`.

Closes: #636

### CODE REVIEW

**Review by ErikSchierboom — APPROVED** [review]
I'm approving as the code owner. I'll let you all decide whether or not the contents are good :)

**Comment by ee7** [comment]
I've tested that this does at least stop suggesting global options with `-n [Tab]` or `--num [Tab]`. So I'll merge. Yell at me if it's broken somehow.

Ideally, we could print something like "specify the number of uuids", instead of making tab completion do nothing. But let's leave that for later.

**Comment by glennj** [comment]
Or we could seed the completions with something like "1", "2", "3", "..." to indicate that an integer is expected.

**Comment by ee7** [comment]
I agree that'd be better. I tried it the time with e.g. 1, 5, and 10. But I gave up when the completions appeared in the order `1 10 5`.

**Pull request merged**
